### ISSUE TITLE
Integratives Projekt

### ISSUE THREAD

**Issue by larafaelivrin** [StataBS/indikatoren#676]
In this private branch the results of the "Integratives Projekt" are plotted.

**Comment by larafaelivrin** [comment]
I could not implement the following things @jb3-2 :

- [ ] 6366: Tooltip is wrongly formated:
![image](https://user-images.githubusercontent.com/23261816/36901799-a7aa819c-1e28-11e8-9fa7-6ef1dfd465c1.png)

- [ ] 6366: *.svg is wrongly formated:
![image](https://user-images.githubusercontent.com/23261816/36901816-b4c60e0a-1e28-11e8-8afe-ea563556c248.png)

- [ ] private => public: Merging conflicts; I do not know if some merging conflicts can be ignored

After this changes have been made this private branch is ready to be merged in to the master branch @jb3-2 @JE1982

**Comment by larafaelivrin** [comment]
@jb3-2 : Can you implement the above mentioned improvements until Wednesday?

**Comment by jb3-2** [comment]
@larafaelivrin You added the same image twice above, is that correct? I only see problems with the sgv of 6363, not 6366.

**Comment by jb3-2** [comment]
@larafaelivrin @JE1982 I have isolated the svg creation problem, but not completely solved it. On Chrome for Mac, the svg is still not nice. On Firefox on Mac, however, it is great. Can you please check how it looks on IE and all the other browsers on Windows? To do that, please open the link below, and search for 6363. Please give me feedback if we can go live like this. 

http://issue-676--retailer-seawater-16683.netlify.com/

**Comment by jb3-2** [comment]
exporting: {allowHTML: true} (see https://api.highcharts.com/highcharts/exporting.allowHTML) solves many issues, but the resulting SVG is still not perfect. Above all, adding the SVG viewBox makes it worse. I have tried Highcharts 6.07 and Export-server 2.0.8, with not much more success. Let's hide this chart in the portal for the moment. 

I have also tried replacing nobreakspace with the unicode u00a0, (see #544), but this did not hep either.
